common/reviews/api/heft.api.md

```diff
@@ -274,6 +274,7 @@ export interface IHeftSessionHooks {
 // @public (undocumented)
 export interface IMetricsData {
     command: string;
+    commandParameters?: Record<string, string>;
```

Comment:
Kinda seems like this should always be defined, and if no parameters are used, it should be `{}`.

Reply:
At the time I think I was trying to be backwards compatible if dependent projects were calling the record function, but you're right I think this is appropriate to define always

Reply:
You can have the parameter in the `record` function be optional and not make this optional.